src/containers/User/UserArea.tsx

```diff
@@ -182,17 +183,23 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({ templates, dataViews }) => {
 
 const BrandArea: React.FC = () => {
   const { strings } = useLanguageProvider()
```

Comment:
this is now unused

Reply:
Yeah okay. I'd left in cos it was used by the commented-out code below, but I've just removed them both now, we don't need it anymore.

Reply:
5ec7d65d